Header header

float32 steering_wheel_angle_cmd #in degrees

bool enabled
bool override
bool timeout
